### PR TITLE
Quotas for Identities API

### PR DESCRIPTION
**Pull request by daniel-almeida-konkconsulting**
# Readiness checklist

- [ ] I added/updated unit tests.
- [X] I added/updated integration tests.
- [X] I ensured that the PR title is good enough for the changelog.
- [X] I labeled the PR.

### CODE REVIEW

File: AdminUi/test/AdminUi.Tests.Integration/AdminUi.Tests.Integration.csproj [hunk 1]
```diff
@@ -40,4 +40,8 @@
     </None>
   </ItemGroup>
 
+  <ItemGroup>
+    <Folder Include="Features\IndividualQuotas\" />
+  </ItemGroup>
```
Comment (tnotheis) [inline AdminUi/test/AdminUi.Tests.Integration/AdminUi.Tests.Integration.csproj:45]: Can be removed (?)

File: AdminUi/test/AdminUi.Tests.Integration/Features/IndividualQuotas/POST.feature [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+Administrator Creates an Individual Quota
+
+Scenario: Creating an Individual Quota for existing Identity
+	Given an Identity i
+	When a POST request is sent to the /Identity/{i.id}/Quotas endpoint
+	Then the response status code is 201 (Created)
+	And the response contains an IndividualQuota
+
+Scenario: Creating an Individual Quota for inexistent Identity
+	When a POST request is sent to the /Identity/{i.id}/Quotas endpoint with an inexistent identity address
```
Comment (tnotheis) [inline AdminUi/test/AdminUi.Tests.Integration/Features/IndividualQuotas/POST.feature:13]:
```suggestion
	When a POST request is sent to the /Identity/{address}/Quotas endpoint with an inexistent identity address
```

**Review by tnotheis — CHANGES_REQUESTED** [review]

File: AdminUi/test/AdminUi.Tests.Integration/AdminUi.Tests.Integration.csproj (hunk 1 above)
Comment (daniel-almeida-konkconsulting) [reply AdminUi/test/AdminUi.Tests.Integration/AdminUi.Tests.Integration.csproj:45]: Yes, not sure why sometimes this gets added automatically.

File: AdminUi/src/AdminUi/Controllers/IdentitiesController.cs [hunk 3]
```diff
@@ -36,4 +39,19 @@ public async Task<IActionResult> GetIdentitiesAsync([FromQuery] PaginationFilter
         var identities = await _mediator.Send(new ListIdentitiesQuery(paginationFilter), cancellationToken);
         return Paged(identities);
     }
+
+    [HttpPost("{identityAddress}/Quotas")]
+    [ProducesResponseType(typeof(IndividualQuotaDTO), StatusCodes.Status201Created)]
```
[the remark this answers is not in the document]
Comment (daniel-almeida-konkconsulting) [reply AdminUi/src/AdminUi/Controllers/IdentitiesController.cs:44]: Copy and paste issue with the TierQuotas endpoint (I'll update that one also).

File: AdminUi/src/AdminUi/Controllers/IdentitiesController.cs [hunk 4]
```diff
@@ -42,6 +42,8 @@ public async Task<IActionResult> GetIdentitiesAsync([FromQuery] PaginationFilter
 
     [HttpPost("{identityAddress}/Quotas")]
     [ProducesResponseType(typeof(IndividualQuotaDTO), StatusCodes.Status201Created)]
+    [ProducesResponseType(StatusCodes.Status404NotFound)]
+    [ProducesResponseType(StatusCodes.Status400BadRequest)]
```
Comment (tnotheis) [inline AdminUi/src/AdminUi/Controllers/IdentitiesController.cs:46]: Use ProducesError

**Review by tnotheis — APPROVED** [review]

**Pull request merged**